client/components/UI/Form/TimeInput/index.tsx

```diff
@@ -53,7 +61,7 @@ export class TimeInput extends React.Component {
                 previousValidValue: '',
                 invalid: false,
                 showLocalValidation: false,
-            });
+            }, this.updateInputWidth);
```

Comment:
You're already calling it in componentDidUpdate - the call here shouldn't be needed.